### ISSUE TITLE
Bitstreamed audio requires toggling audio upmix or passthrough to get audio on first media play

### ISSUE THREAD

**Issue by BigOkie** [koying/SPMC#934]
This is on a new Onkyo receiver and 16.7.0 with Shield TV 2017 [Experience 5.2]

If I start a movie/video with any of the more advanced audio streams (DTS, Dolby, Dolby Plus, etc), then I have to, almost without fail, enter the audio properties within the movie and either toggle stereo upmix or enable passthrough in order for the audio to render.  I'm unsure if this is a SPMC/Android issue or if it's a receiver issue (TX-NR656 Onkyo however it did it intermittently on my old HT-R580 Onkyo) but I know it's pretty consistent.  I'm using all the suggested settings for passthrough from this site, however I did try it using RAW instead of IEC on the passthrough and had the same result.  On subsequent videos being played (in a playlist) this doesn't occur.  However backing out to the home menu resets this and happens again.  Note:  I'm trying something to see if it will fix it (increasing the delay on refresh rate change to 5 seconds sofar seems promising but I will continue to check it).

**Comment by illgodson** [comment]
I have a Vizio tv and Onkyo receiver and I had to set my delay to 6 seconds before it would grab the audio. Even now once in awhile, maybe 1 out of 50 times it will not grab it and I have to start the steam again. I think it's an Onkyo thing have read others with same issue

**Comment by tomatus** [comment]
Hello,

I have exactly the same problem with a Denon receiver. I'm using SPMC v16.6.0 from the play store and set everything to the recommended settings from the wiki.

"Sync playback to display" is disabled and "Adjust display refresh rate" is set between 3 and 6 seconds. I get the error immediately with every movie I start and I need to switch the voice line.

Disabling "Adjust display refresh rate" is fixing the problem, but the result is a worse viewing experience with stutter.

I hope this will be fixed in the next version.

**Comment by illgodson** [comment]
I don't believe it's an issue with SPMC as much as an issue with the receivers/tv combo. I could be wrong, have you tried just keeping adding seconds to the delay until it works? It literally took mine till 6 seconds to work, I ended up upping it to 7 just to be safe

**Comment by tomatus** [comment]
It's working with 8 seconds right now, thanks for the tip.

**Comment by vbaros** [comment]
I have exactly the same issue with my Onkyo reciever. The thing is, it started happening after one of the updates (which happened a long time ago, not recently) of either SMPC or Shield experience.
Before, when it was working, I had the delay set to only one second and it worked every single time without the issue.
